security-admin/src/main/java/org/apache/ranger/rest/ServiceREST.java

```diff
@@ -4298,6 +4308,45 @@ private RangerPolicyList toRangerPolicyList(List<RangerPolicy> policyList, Searc
         return ret;
     }
 
+    private List<RangerPolicy> getRangerPoliciesInRange(List<RangerPolicy> policyList, SearchFilter filter) {
+        List<RangerPolicy> retList = null;
+        if (CollectionUtils.isNotEmpty(policyList)) {
+            int totalCount = policyList.size();
+            int startIndex = filter.getBeginIndex();
+            int pageSize = filter.getOffset();
```

Comment:
Is this(int pageSize = filter.getOffset();) intentional , i think either you have to change the variable name or the call different method.